src/utils/swap.js

```diff
@@ -0,0 +1,227 @@
+/* Logic of processData()
+/* ==============================================
+/* Note: all times are relative to DB's current timestamp. The data fetched from the DB contains a column 'representative_time'.
+/*      This column contains the timestamp that represents the price values of that particular record.
+/*      It is the start of the hour in the case of 'daily', and start of the day in case of 'weekly' or 'monthly' time series.
+/*      In order words, the record fetched will contain the closing values for that particular hour or day.
+/*
+/* 1. We have to calculate the start and end date of the data for the graph.
+/*  End date - for daily, is the (start of the current hour - 1 hour). For weekly/monthly, it is the (start of the current day - 1 day)
+/*  Start date - for daily, it is (End date - 1 day). For weekly, it is (End date - 7 days). For monthly, it is (End date - 31 days).
```

Comment:
Can you please align the indentation of the text on all lines after the first to the same level as the line with the number. For example:
```
 /* 1. We have to calculate the xxx
 /*    End date - for daily, is xxx
 /*    Start date - for daily, xxx
```
Visually it just looks better. Please also do that for the rest of the list.

Reply:
Fixed